packages/dialtone-icons/gulpfile.cjs

```diff
@@ -130,6 +143,57 @@ const buildIcons = function (done) {
   .pipe(dest(paths.icons.outputSvg));
 };
 
+const buildIllustrations = function (done) {
+  //  Make sure this feature is activated before running
+  if (!settings.svgs) return done();
+
+  //  Compile icons
+  return src(paths.illustrations.input)
+    .pipe(replace(' fill="none"', ''))
+    .pipe(replace(' fill="#000"', ' fill="currentColor"'))
+    .pipe(replace(' fill="#000000"', ' fill="currentColor"'))
+    .pipe(replace(' fill="black"', ' fill="currentColor"'))
+    .pipe(replace(' fill="#0D0C0F"', ' fill="currentColor"'))
+    .pipe(replace(' fill="#222"', ' fill="currentColor"'))
+    .pipe(replace(' fill="#222222"', ' fill="currentColor"'))
+    .pipe(replace(/<svg.*(width="[0-9]+|height="[0-9]+)"/g, (match) => {
+      return match
+          .replace(/width="[0-9]+"/, '')
+          .replace(/height="[0-9]+"/, '');
+    }))
+    .pipe(replace('<svg', function (match) {
+      const name = path.parse(this.file.path).name;
+      const title = name
+        .replace(/\b\S/g, t => t.toUpperCase())
+        .replace(/-+/g, ' ');
+      return `${match}
+      aria-hidden="true"
+      focusable="false"
+      role="img"
+      data-name="${title}"`;
+      // class="d-icon d-icon--${name}"
+    }))
+    .pipe(svgmin(function getOptions () {
+      return {
+        multipass: true,
+        plugins: [
+          {
+            removeUnknownsAndDefaults: {
+              keepRoleAttr: true,
+            },
+          },
+          {
+            cleanupIDs: {
+              minify: true,
+            },
+          }
+        ],
+      }
+    }))
+    .pipe(rename({ dirname: '' }))
+  .pipe(dest(paths.illustrations.outputSvg));
+};
```

Comment:
Don't think illustrations need this, this is a pre-processing needed to standardize the icons colors and sizes so we can handle it correctly with CSS.

Please confirm with designers what part of this processing is needed and what is not.

Reply:
I think this isn't needed today, but eventually we will need to have some kind of size for the same illustration, if we want to have it for the future will be great

Reply:
k, I will just leave essential stuff.